traffic_ops/traffic_ops_golang/auth/tenancy.go

```diff
@@ -81,6 +88,9 @@ func IsResourceAuthorizedToUser(resourceTenantID int, user CurrentUser, db *sqlx
 		log.Errorf("Error checking user tenant %v access on resourceTenant  %v: %v", user.TenantID, resourceTenantID, err.Error())
 		return false, err
 	default:
+		if useTenancy == false {
+			return true, nil
+		}
 		if tenantId == resourceTenantID && active == true {
```

Comment:
Change that to `tenantID`..  and not likely to amount to anything, but you could avoid the comparison reversing that condition:
```
if active && tenantID == resourceTenantID ...
```